### ISSUE TITLE
About the choice of pretrained model for downstream tasks

### ISSUE THREAD

**Issue by puzzledsky** [microsoft/Swin-Transformer#84]
It seems that sometimes the model achieve the "Max accuracy" before EPOCH 300 on ImageNet-1K dataset, although their results are in very small difference.

In ths case, which model parameters should be used to train the downstream tasks (e.g. sematic segmentation on ADE20K),  model of Max accuracy or model of EPOCH 300? I want to know your training settings of the paper results.

Thanks!

**Comment by zeliu98** [comment]
Hi, we use the model of the last epoch (epoch 300) for downstream tasks.

**Comment by puzzledsky** [comment]
Thank you for your reply !
